R/chromote.R

```diff
@@ -335,13 +364,14 @@ Chromote <- R6Class(
 
     #' @description Close the [`Browser`] object
     close = function() {
-      if (private$is_active_) {
+      if (self$is_active()) {
+        private$ws$close()
+      }
+      if (self$is_alive()) {
         self$Browser$close()
```

Comment:
I think this ordering of commands won't work properly. `self$Browser` is a domain of the Chrome Devtools Protocol, so `self$Browser$close()` sends the [`Browser.close`](https://chromedevtools.github.io/devtools-protocol/tot/Browser/#method-close) command. But that won't work if the websocket has been closed.

Reply:
Oh yeah, I got confused. I thought this was `self$browser$close()`